src/main/java/seedu/mark/logic/commands/RedoCommand.java

```diff
@@ -15,19 +15,39 @@
 
     public static final String COMMAND_WORD = "redo";
 
-    public static final String MESSAGE_SUCCESS = "Action successfully redone: \"%1$s\"";
-    public static final String MESSAGE_FAILURE = "No more commands to redo!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Redoes a given number of previous undo commands. "
+            + "If not specified, the default number of commands to redo is 1. \n"
+            + "Parameters: [STEP] (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 2";
+
+    public static final String MESSAGE_SUCCESS = "Action(s) successfully redone: \n%1$s";
+    public static final String MESSAGE_FAILURE = "Not enough commands to redo! "
+            + "Only %1$s remaining undo command(s).";
```

Comment:
is this supposed to say redo?

maybe can say "only xxx remaining commands to undo"